sdk/include/opentelemetry/sdk/logs/multi_recordable.h

```diff
@@ -20,6 +19,8 @@ namespace sdk
 {
 namespace logs
 {
+class LogRecordProcessor;
+
```

Comment:
This file has reference to `opentelemetry::common::SystemTimestamp` - seems to be indirectly included. Not a blocker, but can be fixed separately.

Reply:
@lalitb

Good find.

I think we should automate this with include-what-you-use, because doing manual reviews checking all dependencies gets too tedious and error prone.